.github/workflows/ci.yml

```diff
@@ -14,6 +14,22 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
   cancel-in-progress: true
 
+env:
+  FORCE_COLOR: 1 # Request colored output from CLI tools supporting it
+  MYPY_FORCE_COLOR: 1 # MyPy's color enforcement
```

Comment:
I opened https://github.com/python/mypy/issues/13806 which has now been merged in https://github.com/python/mypy/pull/13814, so we can remove this when mypy is next released :)